File: example/scalalib/basic/3-multi-module/bar/test/src/BarTests.scala
package bar
import utest._
object BarTests extends TestSuite {
  def tests = Tests {
    test("simple") {
      val result = Bar.generateHtml("hello")
      assert(result == "<h1>hello</h1>")
      result
    }
    test("escaping") {
      val result = Bar.generateHtml("<hello>")
      assert(result == "<h1>&lt;hello&gt;</h1>")
      result
    }
  }
}

File: example/kotlinlib/script/3-relative-imports/bar/BarTests.kt
//| extends: [mill.script.KotlinModule.Junit5]
//| moduleDeps: [./Bar.kt]
//| mvnDeps:
//| - io.kotest:kotest-runner-junit5:5.9.1
//| - com.github.sbt.junit:jupiter-interface:0.11.2
package bar

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class BarTests :
    FunSpec({

        test("simple") {
            val result = generateHtml("hello")
            result shouldBe "<h1>hello</h1>"
        }

        test("escaping") {
            val result = generateHtml("<hello>")
            result shouldBe "<h1>&lt;hello&gt;</h1>"
        }
    })
